package.json

```diff
@@ -127,7 +127,7 @@
     ]
   },
   "engines": {
-    "node": ">=18 <21"
+    "node": ">=18 <23"
```

Comment:
Node 21 needs to be excluded

Reply:
What about Node 19? I think we should leave it as is since we have a Compatibility table

Reply:
Yes, let's leave Node 19 to not cause a breaking change, but you are correct, it's incorrect as well.

Reply:
Done